terra-batch-libchart/templates/_reverse-proxy.tpl

```diff
@@ -116,7 +116,7 @@ data:
 
         {{- if .Values.wds.coaEnabled }}
         location /wds/ {
-          client_max_body_size 50M;
+          client_max_body_size 500m;
```

Comment:
Assuming that the `M` is not case sensitive?

Reply:
Ah. Apparently not- I got the lowercase m from the nginx documentation for client_max_body_size, I didn't even realize I had changed it!